examples/flows/evaluation/eval-chat-math/line_process.py

```diff
@@ -0,0 +1,43 @@
+from promptflow import tool
+
+
+@tool
+def line_process(groundtruth: str, prediction: str) -> int:

```

Comment:
please add some comments about the cases you may meet of groundtruth and prediction

Reply:
added main function

Reply:
I mean some examples of the input, like for groundtruth what's the possible format you can handle? pure number? and any other format? I ask this because I see a large part of the logic is handle float parse exceptions, so I guess you meet some cases frequently which is not pure numbers. Put some samples values in the comments will be helpful for user understand the function

Reply:
Got it, we'll add some comments in the code line

Reply:
Have added some comments in the code.

Reply:
hi Chenlu and Jieting, thanks for the follow up, the code itself is clear and don't need to comment. I mean you just need to put some examples in comments before the whole logic. For example 
# process prediction 
# will try parse the prediction and groundtruth to float. support parse ones like below: 0.2, 4/10, 1.0/3, 6E-1, 6E-1/0.6 ....